### DISCUSSION TITLE
Integration of good Grepper aggregator tools or add easier ways to use rg etc.

### DISCUSSION THREAD

**Discussion by saumyajyoti** [NvChad/NvChad#1468] [Ideas]
Any plan on enhancing the way we can integrate grep tools e.g. ripgrep. to show search results in qf / loc list? e.g. https://github.com/mhinz/vim-grepper?

**Comment by siduck** [comment]
our telescope live_grep already uses ripgrep by default, however you can other plugins to get your desired behaviour.
